OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -85,7 +85,7 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager)
 				}
 
 				var team = item.Get<LabelWidget>("TEAM");
-				var teamNumber = pp.PlayerReference.Playable ? ((client == null) ? 0 : client.Team) : pp.PlayerReference.Team;
+				var teamNumber = pp.PlayerReference.Playable ? client.Team : pp.PlayerReference.Team;
```

Comment:
See #10878 - it's possible this indicates a missing null check earlier (I don't know enough to say either way).